src/pages/specs/sideloading.tsx

```diff
@@ -41,6 +41,16 @@ export const sideloading = (
       >
         <PreviewScope>
           <P>
+            <Marginale inlineable>
+              <Img
+                src={<ResolveAsset asset={["sideload", "sideload_spot.png"]} />}
+                alt={`An ornamental drawing of various characters transporting and discovering cardboard boxes. Dalton is looking at a map, unaware of the cardboard boxes behind the bush next to them. Alfie is transporting two cardboard boxes on the back of a bicycle. Betty is preparing to launch a cardboard box with a catapult, and seems rather pleased about it..`}
```

Comment:
@sgwilym Should this end with exactly two dots, or is it off by one (either direction makes sense to me)? Those are important quality assurance questions!